gradle-conjure/src/main/java/com/palantir/gradle/conjure/CompileIrTask.java

```diff
@@ -73,11 +85,22 @@ public final void generate() {
                     new File(executableDir.get(), EXECUTABLE).getAbsolutePath(),
                     "compile",
                     inputDirectory.get().getAbsolutePath(),
-                    outputFile.getAbsolutePath());
+                    outputFile.getAbsolutePath(),
+                    "--extensions",
```

Comment:
Is this extensions flag some pre-existing thing (is it part of the spec?)

Reply:
this is something that we are adding here: https://github.com/palantir/conjure/pull/532